doc/developer-guide/cache-architecture/cache-initialization.en.rst

```diff
@@ -0,0 +1,156 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+.. default-domain:: cpp
+.. _cache-initialization:
+
+Cache Initialization
+********************
+
+Initialization starts with an instance of :cpp:class:`Store` reading the storage
+configuration file, by default :file:`storage.config`. For each valid element in
+the file an instance of :cpp:class:`Span` is created. These are of basically
+four types:
+
+* File
+
+* Directory
+
+* Disk
+
+* Raw device
+
+After creating all the :cpp:class:`Span` instances, they are grouped by device
+ID to internal linked lists attached to the :cpp:member:`Store::disk`
+array[#store-disk-array]_. Spans that refer to the same directory, disk, or raw
```

Comment:
The output looks like `array[#store-disk-array]_` in my browser. Is this how it is supposed to look like?

Reply:
Fixed.